docs/install/installation/docker_compose_install.md

```diff
@@ -61,16 +63,10 @@ Prefix commands with `sudo` according to your environment permissions.
 ```bash
 # Stop the container (if running)
 docker-compose down
-# Fetch the latest changes from Github
-git fetch origin
 # Switch to the repo's main branch
 git checkout main
 # Pull the latest changes to the main branch from Github
-git pull origin main
-# Prune all LibreChat Docker images
-docker rmi librechat:latest
-# Remove all unused dangling Docker images
-docker image prune -f
+git pull 
```

Comment:
i would like to keep the old instructions on updating as most used do it the way you revised and and in some edge cases, would run into issues. 

The following extra steps seemed to solve build issues in a lot of cases:

```bash
# Prune all LibreChat Docker images
docker rmi librechat:latest
# Remove all unused dangling Docker images
docker image prune -f
```